Assignment/Cash.c

```diff
@@ -3,12 +3,17 @@
 // Copyright (c) Metamation India.
 // Vinitha Perumal - GET
 // ------------------------------------------------------------------
-// Program.c
-// Program on main branch.
+// Cash.c
+// Program on Test2.2 branch.
 // ------------------------------------------------------------------------------------------------
-#include <stdio.h>
-#include "Program.h"
+#include "Cash.h"
 
-void PrintHelloWorld () {
-	printf ("Hello, World\n");
+int Cash (int cashPaid, int actualAmount, int coins[4]) {

```

Comment:
Your program could be made more dynamic by adding denominations as a parameter. Also, I think it is okay if you directly just send the difference between cost and amount paid through as a param.